src/sentry/integrations/github_enterprise/integration.py

```diff
@@ -312,7 +312,10 @@ def get_installation_info(self, installation_data, access_token, installation_id
         resp = session.get(
             "https://{}/api/v3/user/installations".format(installation_data["url"]),
             params={"access_token": access_token},
```

Comment:
Let's remove the `params` here since you are adding the `access_token` to the headers now